app/assets/javascripts/app/application/uploader.js.coffee

```diff
@@ -101,6 +101,11 @@ convertSize = (str) ->
   else
     null
 
-getFormatsFromFiles = (files) ->
-  if files? && files == 'image/*'
-    ['jpg', 'jpeg', 'png']
\ No newline at end of file
+getFormatsFromAccept = (accept) ->
+  if accept? && accept == 'image/*'
+    ['jpg', 'jpeg', 'png']
+
+# Use this when firefox is ready to use the accepts='.jpg,.png, ...' (version 37 maybe)
```

Comment:
Isn't there a way to check if the current browser supports this and automatically use this option if it does?